packages/luv/luv.0.5.13/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+
+synopsis: "Binding to libuv: cross-platform asynchronous I/O"
+
+license: "MIT"
+homepage: "https://github.com/aantron/luv"
+doc: "https://aantron.github.io/luv"
+bug-reports: "https://github.com/aantron/luv/issues"
+
+authors: "Anton Bachin <antonbachin@yahoo.com>"
+maintainer: "Anton Bachin <antonbachin@yahoo.com>"
+dev-repo: "git+https://github.com/aantron/luv.git"
+
+depends: [
+  "base-unix" {build}
+  "ctypes" {>= "0.14.0"}
+  "dune" {>= "2.7.0"}
+  "ocaml" {>= "4.03.0"}
+
```

Comment:
```suggestion
  "integers" {>= "0.3.0"}
```
It is used transitively from ctypes right now, you call `Unsigned.UInt64.equal` which was introduced in this version

Reply:
Thank you! I'll add this constraint upstream (or switch to `compare`).

Reply:
(for the next version release)